app/src/main/kotlin/com/wire/android/ui/calling/initiating/InitiatingCallScreen.kt

```diff
@@ -43,17 +37,23 @@ fun InitiatingCallScreen(
 ) {
     InitiatingCallContent(
         callState = sharedCallingViewModel.callState,
-        onNavigateBack = { sharedCallingViewModel.navigateBack() },
-        onHangUpCall = { sharedCallingViewModel.hangUpCall() }
+        toggleMute = sharedCallingViewModel::toggleMute,
+        toggleVideo = sharedCallingViewModel::toggleVideo,
+        onNavigateBack = sharedCallingViewModel::navigateBack,
+        onHangUpCall = sharedCallingViewModel::hangUpCall,
+        onVideoPreviewCreated = { sharedCallingViewModel.setVideoPreview(it) }
```

Comment:
`onVideoPreviewCreated = sharedCallingViewModel::setVideoPreview` or keep lambda if you clearly want it to be recomposed every time 🤔

Reply:
Yes but I am passing a parameter to it

Reply:
as long as parameters and return types match then it can be used this way, and in this case I think both are the same so I think it can be `onVideoPreviewCreated = ::setVideoPreview` 😉